application/account-management/WebApp/src/lib/applicationInsights/config.tsx

```diff
@@ -0,0 +1,25 @@
+import { ReactPlugin } from "@microsoft/applicationinsights-react-js";
+import { ApplicationInsights } from "@microsoft/applicationinsights-web";
+
+export const reactPlugin = new ReactPlugin();
+
+const applicationInsights = new ApplicationInsights({
+  config: {
+    appId: "account-management/webapp",
+    instrumentationKey: "webapp",
+    disableInstrumentationKeyValidation: true,
+    endpointUrl: "/api/track",
+    enableAutoRouteTracking: true,
+    autoExceptionInstrumented: true,
+    autoUnhandledPromiseInstrumented: true,
+    autoTrackPageVisitTime: false,
+    isBeaconApiDisabled: true,
+    extensions: [reactPlugin],
+    disableFetchTracking: true,
+    disableAjaxTracking: true,
+  },
+});
+
+// Load the Application Insights script
```

Comment:
What I meant by documentation was the settings above like `isBeaconApiDisabled`, `disableFetchTracking`, etc., and specifically why we don't just use the defaults.